ocs_ci/ocs/ui/base_ui.py

```diff
@@ -87,12 +87,14 @@ def choose_expanded_mode(self, mode, by_locator, type):
             self.do_click(by_locator=by_locator, type=type)
 
 
-def login_ui(browser):
+def login_ui(browser, headless=True, chrome_type=ChromeType.CHROMIUM):
     """
     Login to OpenShift Console
 
     Args:
         browser(str): type of browser (chrome, firefox..)
+        headless(bool): True- without GUI, False- with GUI
+        chrome_type (ChromeType): Select chrome type (chrome, chromium..)
```

Comment:
The `chrome_type` seems to can be one of those:
- `chromium`
- `google-chrome`
- `edge` (probably not applicable for us)
so I would list those items exactly - to help people distinguish, what value directly use.

Also initially I meant to make those parameters configurable via configuration file.
(Like for example used here for `disconnected` parameter: https://github.com/red-hat-storage/ocs-ci/blob/4bc1c51cd7b9e97ae2d6b42e8936aea2eec1724c/ocs_ci/helpers/helpers.py#L1090
Second parameter will be the default `ChromeType.CHROMIUM`.
It will probably make sense to put it to different section than `DEPLOYMENT`.
It should be then also listed here: https://github.com/red-hat-storage/ocs-ci/blob/master/conf/README.md